sdk/storage/Azure.Storage.Blobs/src/Models/BlobDownloadResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// The details and Content returned from downloading a blob.
```

Comment:
It doesn't have to happen in this PR, but we should spend a little more time on the doc comments in these descriptions.  It'd be awesome to pull a few test code snippets into our ref docs of getting the data as a string and getting the data as JSON since that's what so many customers will want to do.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/19271